scripts/restore.sh

```diff
@@ -0,0 +1,120 @@
+#!/bin/bash
+
+# Backup file directory path
+BACKUP_DIRECTORY_PATH="/palworld/backups"
+
+# Resotre path
+RESTORE_PATH="/palworld/Pal"
+
+# Copy the save file before restore temporary path
+TMP_SAVE_PATH="/palworld/backups/restore-"$(date +"%Y-%m-%d_%H-%M-%S")
+
+# shellcheck disable=SC2317
+term_error_handler() {
+    echo "An error occurred during server shutdown."
+    exit 1
+}
+
+# shellcheck disable=SC2317
+restore_error_handler() {
+    printf "\033[0;31mAn error occurred during restore.\033[0m\n"
+    if [ -d "$TMP_SAVE_PATH/Saved" ]; then
+        read -rp "I have a backup before recovery can proceed. Do you want to recovery it? (y/n): " RUN_ANSWER
+        if [[ $RUN_ANSWER == "y" ]] || [[ $RUN_ANSWER == "Y" ]]; then
```

Comment:
I'd convert to lowercase for comparison.
```suggestion
        if [[ ${RUN_ANSWER,,} == "y" ]]; then
```